mopidy/audio/tags.py

```diff
@@ -53,6 +53,14 @@ def convert_taglist(taglist):
                 result[tag].append(value.decode('utf-8', 'replace'))
             elif isinstance(value, (compat.text_type, bool, numbers.Number)):
                 result[tag].append(value)
+            elif isinstance(value, Gst.Sample):
+                buf = value.get_buffer()
```

Comment:
https://lazka.github.io/pgi-docs/#Gst-1.0/structs/Sample.html#Gst.Sample.get_buffer says `get_buffer()` can return `None`, code should probably check for that.